sharkfuser/tests/lit/test_conv_asm_emitter_ndhwc_kdrsc_grouped.cpp

```diff
@@ -9,8 +9,6 @@
 // XFAIL: {{.*}}
 // RUN: %{TEST_EXE} | iree-opt --verify-roundtrip
 // RUN: %{TEST_EXE} | FileCheck %s --check-prefix=TORCH-CHECK
-// RUN: %{TEST_EXE} | iree-compile - --compile-to=input | \
-// RUN:             FileCheck %s --check-prefix=LINALG-CHECK
```

Comment:
Do we need then update comment `TODO` above (remove the asking about adding of `LINALG-CHECK section`)?